tests/Unit/CartTest.php

```diff
@@ -52,4 +53,16 @@ public function testGetItemsCount()
         $cart->getItems();
         $this->assertEquals(3, count($cart->getItems()));
     }
+
+    /**
+     * Render the cart HTML table
+     *
+     * @covers Cart::getItems
+     */
+    public function testRenderHtml()
```

Comment:
@AC-TimRourke 
When I run the test, it's giving me this error message but I'm not sure why it's saying matching type isn't a `string`

```php
CartTest::testRenderHtml
Array () does not match expected type "string".
```